src/main/java/org/broadinstitute/consent/http/service/LibraryCardService.java

```diff
@@ -124,4 +128,62 @@ private void throwIfNull(LibraryCard libraryCard) {
             throw new NotFoundException("LibraryCard not found.");
         }
     }
+
+    //helper method for create method, checks to see if card already exists
+    private void checkIfCardExists(LibraryCard payload) throws Exception {
+        Integer userId = payload.getUserId();
+        String email = payload.getUserEmail();
+        Integer institutionId = payload.getInstitutionId();
+        Optional<LibraryCard> foundCard;
+        List<LibraryCard> results;
+
+        if(Objects.nonNull(payload.getUserId())) {
+            results = libraryCardDAO.findLibraryCardsByUserId(userId);
+        } else if(Objects.nonNull(email)) {
+            results = libraryCardDAO.findAllLibraryCardsByUserEmail(email);
+        }
+        else {
+            throw new BadRequestException();
+        }
+        foundCard = results.stream().filter(card -> {
+            Boolean sameUserId = Objects.nonNull(userId) && card.getUserId().equals(userId);
+            Boolean sameUserEmail = Objects.nonNull(email) && card.getUserEmail().equalsIgnoreCase(email);
+            Boolean sameInstitution = card.getInstitutionId().equals(institutionId);
+            return (sameUserId || sameUserEmail) && sameInstitution;
+        }).findFirst();
+
+        if(foundCard.isPresent()) {
+            //This exception is a placeholder, what's an exception that can be translated as "CONFLICT"
+            throw new ConsentConflictException();
+        } 
+    }
+
+    //Helper method to process user data on create LC payload
+    //Needed since CREATE has a unique situation where admins can create LCs without an active user (save with userEmail instead)
+    private LibraryCard processUserOnNewLC(LibraryCard card) {
+        if (Objects.isNull(card.getUserId())) {
+            if (Objects.isNull(card.getUserEmail())) {
+                throw new BadRequestException();
+            } else {
+                try{
+                    //If a user is found, update the card to have the correct userId associated
```

Comment:
instead of doing this in a try catch, maybe you could just check if the user is null before getting and setting the user id since the only exception being thrown here would be a null pointer